SIRO_README.md

```diff
@@ -22,22 +22,34 @@ Project-specific README for SIRo.
 
 # Floorplanner Dataset
 
-FP is actually four distinct pieces: Floorplanner scenes, Amazon-Berkeley objects, Google Scanned objects, and Floorplanner episodes.
+FP is actually five distinct pieces: (1) Floorplanner scenes, (2) Amazon-Berkeley objects, (3) Google Scanned objects, (4) Floorplanner scenes, and (5) Floorplanner episodes. (assuming you already have ycb objects)
 
-1. Download Floorplanner Scenes: [fpss_osmm.zip](https://drive.google.com/file/d/1-utUMfUbbzg_zUE5GcGNdk1UEK6lSbXe/view?usp=sharing)
+1. Download Floorplanner Scenes:
+```
+# Go to the habitat data directory
+cd habitat-lab/data
+# Make sure you have git-lfs installed
+git lfs install
+# Clone dataset, and it will take a while to finish
+git clone https://huggingface.co/datasets/fpss/fphab
+# change the name, and Floorplanner Scenes is ready to use
+mv fphab fpss
```

Comment:
Is this really necessary? Can we instead update our references to `fpss` to point to the correct `fphab`?

Reply:
Hi Eric! thanks this. I think this is necessary as when I generated the dataset, I used `fpss` due to the fact that it was `fpss` before. If we change this, we need to re-generate the dataset. I think we can keep this as it is for now, and I can generate the dataset using `fphab`

Reply:
Alternatively, we could instruct to create a symlink to the dataset. This also allows for sharing the data with multiple clones, which I assume is a common setup.

`ln -s /path/to/fpss/fphab data/fpss`